src/state/lists/utils.test.ts

```diff
@@ -0,0 +1,64 @@
+import { TokenList, VersionUpgrade } from '@uniswap/token-lists'
+
+import { shouldAcceptMinorVersionUpdate } from './utils'
+
+function buildTokenList(count: number): TokenList {
+  const tokens = []
+  for (let i = 0; i < count; i++) {
+    tokens.push({
+      name: `Token ${i}`,
+      address: `0x${i.toString().padStart(40, '0')}`,
+      symbol: `T${i}`,
+      decimals: 18,
+      chainId: 1,
+      logoURI: `https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/ethereum/assets/0x${i
+        .toString()
+        .padStart(40, '0')}/logo.png`,
+    })
+  }
+  return {
+    name: 'Defi',
+    logoURI:
+      'https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/ethereum/assets/0x514910771AF9Ca656af840dff83E8264EcF986CA/logo.png',
```

Comment:
nit: empty line?

Reply:
i'm confused what you mean - this is automatic formatting

Reply:
Yeah, I guess I am confused by the fact automated formatted did an empty line. But since it did, I will skip that.